apps/site/src/views/Home/sections/FAQ/FAQ.module.scss

```diff
@@ -0,0 +1,121 @@
+@use "zothacks-theme" as theme;
+@use "bootstrap-utils" as utils;
+
+.faq {
+	position: relative;
+	display: flex;
+	flex-direction: column;
+
+	margin-left: 40px;
+	margin-right: 40px;
+	padding-bottom: 30px;
+
+	background-color: theme.$beige;
+}
+
+.title {
+	align-self: center;
+	margin-top: 24px;
+	margin-bottom: 24px;
+}
+
+.vertical-line {
+	position: absolute;
+	justify-self: flex-start;
+
+	height: 100%;
+	width: 116px;
+	border-right: 4px solid theme.$red;
+	opacity: 0.45;
+}
+
+.body {
+	margin-left: 110px;
+	margin-right: 10px;
+}
+
+.accordion {
+	--bs-accordion-bg: transparent;
+	--bs-accordion-active-bg: transparent;
+	--bs-accordion-border-color: transparent;
+	--bs-accordion-btn-focus-box-shadow: 0;
+	--bs-accordion-border-radius: 0;
+	--bs-accordion-btn-icon-width: 14px;
```

Comment:
Praise: nice work overriding the Bootstrap properties.
Issue: we should still include the focus box shadow for accessible navigation.

Reply:
Given the structure of the current css, the box shadow looks quite off. I'm sure I would be able to stye it but I'm unsure of how it would look given the current figma design
![Screenshot 2023-10-02 at 8 21 24 PM](https://github.com/HackAtUCI/zothacks-site-2023/assets/81263982/91f4727d-98ff-43f1-98bb-90e7cae7b860)

Reply:
Maybe we could try to align the left edge better, change the color, or use a different effect on focus? Feel free to consult graphics as well.

Reply:
For context, we did not use a box shadow for this same part in the Hack at UCI 2023 website. If necessary, however, we can resolve this in a new issue for the sake of time.